### ISSUE TITLE
Welcome message not scrollable - needed when not fitting on the screen

### ISSUE THREAD

**Issue by HenrikBengtsson** [MarcusWolschon/osmeditor4android#3021]
## Vespucci Version

```
Vespucci 21.1.5
Flavor: current
Target SDK: 35
Maximum avaliable memory 536870912
Total memory used (non-native) 124895936
Native memory usage 59578736
In memory Tile Cache MAPNIK type tms tiles BITMAP usage Size 7864320 of maximum 67108864 #entries 30
Selection stack
Selection 0 0 nodes 0 ways 0 relations
Current used file system net tile cache size: 269593B
State file size 4654 last changed 2025-11-04 20:59:23
Bug state file size 60 last changed 2025-11-04 20:59:23
Relations (current/API): 0/0
Ways (current/API): 0/0
Nodes (current/Waynodes/API): 0/0/0
Available location providers
passive enabled true
network enabled true
fused enabled true
gps enabled true
Configured layers
IMAGERY 0 IMAGERY true MAPNIK
SCALE 1 SCALE true null
OSMDATA 2 OSMDATA true null
TASKS 3 TASKS true null
```

## Device (Manufacturer and Model)

Google Pixel 8a

## Android Version 

Android 14

## Behaviour/Symptoms

Cannot see full Welcome message;

<img width="420" height="933" alt="Image" src="https://github.com/user-attachments/assets/5d790732-36b8-4485-b3d5-5c636bf9726a" />

## Expected Behaviour

A scrollbar.

## How to recreate

Increase font size in Android. Clear data for app and restart app.

## Crash dump submitted (no or yes + date)

N/A
